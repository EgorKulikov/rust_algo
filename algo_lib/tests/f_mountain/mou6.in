20000
I 352 16862 49988
I 8769 11688 49871
Q 599319665
I 13318 19009 50015
I 10089 12287 50028
Q 831853897
I 16120 19212 50010
I 5150 8199 49863
Q 307350408
I 7995 8828 50203
I 15207 17465 49703
Q 773685396
I 577 2230 50164
I 4920 6323 50174
Q 573194281
I 14517 15158 49576
I 7107 8125 50805
Q 765996671
I 13325 13361 52720
I 15786 17096 50561
Q 848006569
I 1598 2859 50499
I 19572 19882 49721
Q 957855289
I 9953 10533 49442
I 11637 12133 50360
Q 938095728
I 2337 3322 49584
I 7390 7801 49818
Q 918163782
I 2659 3434 50653
I 12663 12692 52260
Q 727805270
I 15525 15618 50043
I 7892 8392 51365
Q 473292358
I 14341 14864 48886
I 6417 6842 51073
Q 874790252
I 19509 19657 52694
I 8009 8662 51087
Q 732059398
I 3785 4285 48818
I 1477 1649 49027
Q 637585430
I 3002 3506 51040
I 11627 11826 50985
Q 839593480
I 9397 9537 48520
I 6175 6419 48805
Q 944908967
I 18212 18416 52883
I 3223 3232 70162
Q 579478216
I 16535 17003 50519
I 16115 16553 51095
Q 873121775
I 2066 2468 48840
I 15891 16080 52634
Q 946999490
I 14481 14759 48470
I 15372 15405 41184
Q 799145556
I 3615 3617 -206072
I 13832 14189 50414
Q 777549368
I 17543 17798 50846
I 7610 7881 50674
Q 845472191
I 7507 7884 50058
I 11953 12168 50649
Q 856190864
I 19544 19844 47818
I 3048 3273 49918
Q 504235768
I 7330 7645 51665
I 15083 15422 48459
Q 948653780
I 1193 1284 46367
I 16584 16779 50611
Q 886633285
I 3941 4032 41249
I 13475 13535 54700
Q 687278194
I 1480 1516 38848
I 3194 3230 27977
Q 293473705
I 2961 3116 51725
I 8326 8407 56545
Q 564505954
I 9429 9471 65904
I 19499 19599 47124
Q 954046896
I 5195 5358 52569
I 8899 9111 51681
Q 560000761
I 19219 19394 50263
I 8422 8609 46597
Q 689817122
I 7233 7265 51020
I 7247 7385 48849
Q 700734812
I 5962 6223 49092
I 16560 16768 48884
Q 898865410
I 12141 12159 56706
I 11907 11959 40904
Q 879904289
I 991 1233 49335
I 18940 19192 49460
Q 931427323
I 5574 5825 47261
I 16844 16948 48941
Q 925346747
I 6414 6603 48664
I 18426 18627 50935
Q 960092069
I 12219 12317 44554
I 17959 17979 39839
Q 933019342
I 3200 3206 99113
I 2484 2703 49026
Q 221214879
I 7440 7459 43197
I 18389 18560 47380
Q 946975106
I 14659 14707 56270
I 11285 11468 53174
Q 877989999
I 19467 19522 49736
I 12708 12871 47530
Q 936168036
I 7454 7545 55627
I 15281 15340 54025
Q 891473495
I 3426 3616 52414
I 13331 13410 47110
Q 712913526
I 14333 14375 56237
I 7875 8079 50586
Q 966246677
I 16167 16235 39636
I 7304 7361 43825
Q 507256884
I 1668 1791 54287
I 3539 3628 49473
Q 737979805
I 400 499 54713
I 16776 16973 51199
Q 914278145
I 14829 14830 -133617
I 9840 9854 91136
Q 877537104
I 9882 9985 54157
I 19593 19708 57049
Q 965041164
I 16344 16395 47740
I 683 767 43473
Q 340053111
I 15443 15601 46666
I 16196 16346 46597
Q 930191839
I 18044 18065 75971
I 14040 14169 55609
Q 970984337
I 19540 19617 58703
I 19232 19407 52248
Q 966129243
I 16547 16633 46167
I 16143 16234 51775
Q 841753943
I 14638 14804 49218
I 12603 12624 36305
Q 881605495
I 15013 15037 66920
I 18842 18976 49325
Q 940326286
I 10178 10184 48731
I 1677 1770 49208
Q 183438435
I 8184 8323 48907
I 6931 6998 61364
Q 700737100
I 6445 6448 126986
I 13384 13479 54068
Q 661661309
I 13183 13207 48682
I 19531 19557 64945
Q 973035950
I 1343 1426 50365
I 240 317 43668
Q 906658434
I 8175 8254 42604
I 33 56 74033
Q 752090098
I 4023 4078 50168
I 18046 18059 51490
Q 917905412
E
