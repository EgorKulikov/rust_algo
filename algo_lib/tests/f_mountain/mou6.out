12347
16991
6502
15835
11808
15654
17279
19827
19074
18697
14880
9778
17802
14952
13069
17101
19515
11908
17776
19539
16316
15911
17242
17459
10443
19590
18072
14096
6288
11669
19709
11550
14144
14368
18333
17969
18989
18885
19707
19060
4835
19355
17947
19122
18214
14640
19784
10526
15136
18649
17907
19718
7197
18982
19846
19710
17194
18005
19160
4057
14357
13580
19832
18393
15291
18604
